### PR TITLE
Feat: Add EIP712 sigining support to ledger

### LINKED ISSUE #538: Support EIP-712 sigining in Ledger
EIP-712 signing is introduced in `5.27.0` and onboard uses `5.21.0`.

After that need to add a provision to support EIP712 signing.

### PR DESCRIPTION
**Pull request by koushikmohan1996**
Fixes #538 

Similar implementation is done in ledger live app - https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/ethereum/hw-signMessage.js#L43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,15 +46,15 @@
     "prettier": "^2.0.5",
     "rimraf": "^3.0.0",
     "rollup": "^1.27.5",
-    "rollup-plugin-svelte": "^5.0.3",
+    "rollup-plugin-svelte": "^6.1.1",
```
Comment (koushikmohan1996) [inline package.json:49]: Refer this https://github.com/sveltejs/svelte/issues/5665

The package is not building for me in the local environment. If it's not needed I will revert this.

Comment (koushikmohan1996) [reply package.json:49]: @taylorjdawson 
Maybe I am doing something wrong? All I did is clone, `npm install` and `npm build`. Do I have to configure something else?

Comment (taylorjdawson) [reply package.json:49]: @koushikmohan1996  What node and npm version are you using?

Comment (koushikmohan1996) [reply package.json:49]: @taylorjdawson  npm - 6.14.5, node - 12.18.2

Comment (taylorjdawson) [reply package.json:49]: Hmm Also not sure if this would make a difference but have you tried building with yarn?

Comment (koushikmohan1996) [reply package.json:49]: @taylorjdawson Weird. That actually fixed the problem. I will revert that. You can review the rest of the changes

Comment (taylorjdawson) [reply package.json]: Nice!

**Comment by taylorjdawson** [comment]
@koushikmohan1996 Could you outline the steps you took to test this so I can repro locally? Thanks!

**Comment by koushikmohan1996** [comment]
> @koushikmohan1996 Could you outline the steps you took to test this so I can repro locally? Thanks!

@taylorjdawson  So basically provider should respond to the `eth_signTypedData` RPC call. I tested it in Opyn where we sign limit order with EIP712. Maybe I can host a demo site

**Comment by taylorjdawson** [comment]
> > @koushikmohan1996 Could you outline the steps you took to test this so I can repro locally? Thanks!
> 
> @taylorjdawson So basically provider should respond to the `eth_signTypedData` RPC call. I tested it in Opyn where we sign limit order with EIP712. Maybe I can host a demo site

That would would super helpful, thank you!

**Pull request closed without merging**